internal/pkg/aws/regions/regions.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package regions
+
+import (
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/copilot-cli/internal/pkg/aws/partitions"
+)
+
+// IsServiceAvailable returns true if the service ID is available in the given region.
+func IsServiceAvailable(sid string, region string) (bool, error) {
```

Comment:
nits:
```suggestion
func IsAvailableInRegion(sID string, region string) (bool, error) {
```
Here and below: can we make the function names more declarations than questions, for consistency, and because they're returning a booleans?